components/builder-web/app/origin/origin-page/origin-job-detail/_origin-job-detail.component.scss

```diff
@@ -0,0 +1,78 @@
+.origin-job-detail-component {
+  .jobgroup-overview {
+    @include shadow-panel;
+
+    section.jobgroup-details {
+      margin-top: 0px;

```

Comment:
Minor, but `0px` (or `0` pretty-much-anything) should always just be `0` -- the linter should be catching these, though. 😕